webkit2gtk: update to 2.40.3.

magackame opened the pull request:
[ci skip]

Fixes `nyxt` package.

#### Testing the changes
- I tested the changes in this PR: **YES**

Tested using `luakit`, `nyxt` and `/usr/libexec/webkit2gtk-4.0/MiniBrowser` - all works.

#### Local build testing
- [x] I built this PR locally for my native architecture, x86_64-glibc
- [x] I built this PR locally for these architectures:
  - [x] armv6l

oreo639 commented:
https://github.com/WebKit/WebKit/commit/7093bcb6d649597962c82fd0e9a92d329ba8fb48

You need to define `-DBWRAP_EXECUTABLE` and `-DDBUS_PROXY_EXECUTABLE`. (it should just be the normal /usr/bin path)

magackame commented:
I added the suggested options and now armv6l build works!

Updated the initial commit message.

Also, do I need to modify `common/shlibs` or anything else that I maybe missed?

oreo639 commented:
> Also, do I need to modify `common/shlibs`?

Not unless a shlib was added or updated, which shouldn't be the case for patch versions.
